src/neo/NeoSystem.cs

```diff
@@ -36,6 +37,7 @@ public NeoSystem(string storageEngine = null)
             this.Blockchain = ActorSystem.ActorOf(Ledger.Blockchain.Props(this, store));
             this.LocalNode = ActorSystem.ActorOf(Network.P2P.LocalNode.Props(this));
             this.TaskManager = ActorSystem.ActorOf(Network.P2P.TaskManager.Props(this));
+            this.SyncManager = ActorSystem.ActorOf(Network.P2P.SyncManager.Props(this));
```

Comment:
Omg, SyncManager? aheuaheuaheuaea
Specialized in syncing a node up-to-date?

Sounds like a subtask of TaskManager.

Reply:
> Sounds like a subtask of TaskManager.

Yes, we're trying to use block index for synchronization 😂